### PR TITLE
[ACA-2775] e2e remove some redundant awaits

### PR DESCRIPTION
**Pull request by adinapitul**
- remove some awaits
- add try catch
- small refactoring

Issue Number: ACA-2775

### CODE REVIEW

File: e2e/components/breadcrumb/breadcrumb.ts [hunk 1]
```diff
@@ -45,11 +45,11 @@ export class Breadcrumb extends Component {
   }
 
   async getNthItemName(nth: number) {
```
Comment (DenysVuika) [inline e2e/components/breadcrumb/breadcrumb.ts]: Please add return types to functions in this file

File: e2e/components/data-table/data-table.ts [hunk 2]
```diff
@@ -104,7 +104,7 @@ export class DataTable extends Component {
 
   async getColumnHeadersText() {
```
Comment (DenysVuika) [inline e2e/components/data-table/data-table.ts]: missing return types in functions

**Comment by adinapitul** [comment]
@DenysVuika , return types are missing almost everywhere. I would like to address that in a separate PR and separate JIRA task.

**Comment by DenysVuika** [comment]
Makes sense, thanks

**Pull request merged**